### DISCUSSION TITLE
Upgrade helper is not working ?

### DISCUSSION THREAD

**Discussion by adsalihac** [react-native-community/discussions-and-proposals#658] [Ideas]
i am looking upgrade helper website for 1/2 weeks. they version name list is still loading , any idea about that ? 

https://react-native-community.github.io/upgrade-helper/

sorry for added this issue in idea section . if possible add `issue discussion` section.

**Comment by cortinico** [comment]
Please report issues to the upgrade helper here:
- https://github.com/react-native-community/upgrade-helper/

**Reply by adsalihac** [reply]
already done , but no reply from anyone
